src/details/ArborX_DetailsTreeNodeLabelling.hpp

```diff
@@ -22,64 +22,69 @@ namespace Experimental
 {
 
 template <class ExecutionSpace, class BVH, class LabelsIn, class LabelsOut>
-void initBVHLabels(ExecutionSpace const &exec_space, BVH const &bvh,
-                   LabelsIn const &in, LabelsOut &out)
+void initLabels(ExecutionSpace const &exec_space, BVH const &bvh,
+                LabelsIn const &in, LabelsOut &out)
 {
   auto const n = bvh.size();
 
+  ARBORX_ASSERT(n >= 2);
```

Comment:
No, I don't think this should be here. `n=1` is a valid tree, so `out` should be set in that case.

Reply:
Fine with updating